package.json

```diff
@@ -86,6 +91,9 @@
     "atLeast": 100,
     "detail": true,
     "strict": true,
-    "ignoreCatch": true
+    "ignoreCatch": true,
+    "ignoreFiles": [
+      "index.d.ts"
```

Comment:
Needed for what we copied in from `hast-util-to-mdast`, can be removed when that’s updated